README.md

```diff
@@ -192,6 +192,76 @@ In order to package libraries when connected to an Orchestrator instance, ensure
 
 Do you have any questions regarding the plugin? Ask them [here](https://connect.uipath.com/marketplace/components/jenkins-plugin-for-uipath-public-preview/questions).
 
+## Troubleshooting
+#### [Unauthorized error](#markdown-unauthorized-error)
```

Comment:
Hmm, in preview mode the links don't seem to work for some odd reason, but I guess it's just a preview rendering issue.